src/modules/attributes.ts

```diff
@@ -28,18 +30,16 @@ function updateAttrs(oldVnode: VNode, vnode: VNode): void {
         elm.setAttribute(key, "");
       } else if (cur === false) {
         elm.removeAttribute(key);
+      } else if (key.charCodeAt(3) === colonChar) {
```

Comment:
I think we should preserve the initial `key.charCodeAt(0) !== xChar` from before. It creates a fast path for the common case where no special namespace is needed and also makes the colon checks slightly safer as an inial "x" is then known.